adding refresh request behavior (#10339)
opensearch-project/documentation-website: _api-reference/index-apis/refresh.md

To use the Refresh Index API, you must have write access to the indexes you want to refresh.

## Refresh request behavior

> The Refresh Index API call is synchronous. The response is only returned after refreshing all targeted shards.

Reviewer: ```suggestion
The Refresh Index API call is synchronous. The response is returned only after all targeted shards have been refreshed.
```

> Because refresh operations are resource-intensive, we recommend relying on automatic periodic refreshes using `index.refresh_interval`.

Reviewer: ```suggestion
Because refresh operations are resource intensive, we recommend relying on automatic periodic refreshes using `index.refresh_interval`.
```

> Because refresh operations are resource-intensive, it is generally recommended to rely on automatic periodic refreshes using `index.refresh_interval`.

Reviewer 2: ```suggestion
Because refresh operations are resource-intensive, we recommend relying on automatic periodic refreshes using `index.refresh_interval`.
```
